src/cryptonote_core/blockchain.cpp

```diff
@@ -2372,7 +2372,7 @@ bool Blockchain::check_tx_inputs(transaction& tx, uint64_t& max_used_block_heigh
   TIME_MEASURE_FINISH(a);
   if(m_show_time_stats)
```

Comment:
These blocks of code are almost identical, but not quite. If they are meant to show the same information, it might make sense to deduplicate it in a maybe_show_time_stats(<tx>) function that has it all in one place. Easier to maintain at least.